dev-cli/container/src/ao-build-module

```diff
@@ -55,19 +64,28 @@ def main():
         c_program = c.read()
 
     link_libraries = []
-    dependencies_libraries = []
-    # Inject c files into c_program if language is c
     c_source_files = []
+    injected_lua_files = []
+    dependency_libraries = []
+    
+    # Inject c files into c_program if language is c
     if(language == 'c'):
-        c_program = inject_c_files(definition, c_program, c_source_files, link_libraries)
+        c_program = inject_c_files(definition, c_program, c_source_files)
```

Comment:
This PR removed the fourth parameter `link_libraries` from this function call, without removing it from the function definition, causing #1205.